docs/migration-guides/v4/color-palette.md

```diff
@@ -1,6 +1,6 @@
 # Deprecated Color Variables
 
-In V4, we are using Spark tokens for colors and will no longer be exporting the following color palette variables. Below is a list of color variables and their hex equivalents.
+In V4, we are using Spark tokens for colors and will no longer be exporting the following color palette variables. Below is a list of color variables and their hex equivalents. Read more about the new Spark 4.0 Design System Color palette on the [Spark Design System documentation site](https://spark.genesys.com/latest/foundations/color/core-palette/design-XYztDeyk)
 
 ## Blue
```

Comment:
Perhaps capitalizing `Variable` and `Value` in the table headers in this file.

<img width="254" alt="Screen Shot 2023-11-02 at 11 04 41 AM" src="https://github.com/MyPureCloud/genesys-spark/assets/127438502/ab73b936-67ff-484e-808f-fa0131d0e49d">